README.md

```diff
@@ -154,7 +154,7 @@ scoop "https://lil.law.harvard.edu" --capture-timeout 90000 --load-timeout 60000
   <summary><strong>See: Output of scoop --help 🔍</strong></summary>
 
 ```
-Usage: scoop [options] <url>
+Usage: @harvard-lil/scoop [options] <url>
```

Comment:
Is this right? Can you run on the command line with the `@harvard-lil/` prefix? It's not working in the VM I'm fussing with.

Reply:
It's not. Was automatically added from `package.json`, will fix! Good catch :D